http_server/printer_seq.py

```diff
@@ -0,0 +1,57 @@
+from amaranth import Module, Signal
+from amaranth.lib.wiring import connect
+
+from printer import AbstractPrinter
+from stream_mux import StreamMux
+
+class PrinterSeq(AbstractPrinter):
+    """
+    When activated, activates sub-printers in sequence.
+
+    Does not claim the sequence of printers as submodules, so they 
+    must be owned by something else.
+
+    AbstractPrinter Attributes
+    ----------
+    output: Stream(8), out
+            The data stream to write the message to.
+    en:     Signal(1), in
+            One-shot trigger; start writing the message to output.
+    done:   Signal(1), out
+            High when inactive, i.e. writing is done.
+    """
+
+    def __init__(self, sequence):
+        import math
+        self._sequence = sequence
+        super().__init__()
+
+    def elaborate(self, _platform):
+        m = Module()
+
+        m.d.comb += [
+            self.done.eq(self._sequence[-1].done),

```

Comment:
This is OK for hacking on, but if we were engineering I'd have a concern here:

`Printer`, at least, assumes `en` is a pulse: if `en` is held high, it'll keep repeating. That's ~okay for `Printer`, because it'll just be blocked on the output channel.

But something that latches on the `~en -> en` edge -- like my earlier naive digit printer -- would re-latch at the *end* of its print cycle, rather than at the start.

If we're chaining `en` and `done`, we should make them consistent- either `done` is a pulse, or `AbstractPrinter`s use the edge of `en` rather than its level.